service/dap/server_test.go

```diff
@@ -2713,8 +2713,14 @@ func TestWorkingDir(t *testing.T) {
 					client.VariablesRequest(1001) // Locals
 					locals := client.ExpectVariablesResponse(t)
 					checkChildren(t, locals, "Locals", 2)
-					checkVarExact(t, locals, 0, "pwd", "pwd", fmt.Sprintf("%q", wd), "string", noChildren)
-					checkVarExact(t, locals, 1, "err", "err", "error nil", "error", noChildren)
+					for i := range locals.Body.Variables {
```

Comment:
Is this because they may not be present? We switched from always checking this to only if the name we care about is in the list of `locals.Body.Variables`, correct?

Reply:
No, it's because the order isn't guaranteed, and it changed in 1.17.